libvirt/tests/src/migration/migration_with_stress/network_load_on_host.py

```
@@ -0,0 +1,87 @@
+        process.run("pkill netserver", shell=True, verbose=True, ignore_status=True)
+        remote.run_remote_cmd("pkill netserver", params, ignore_status=True)
+        process.run("pkill netperf", shell=True, verbose=True, ignore_status=True)
+        remote.run_remote_cmd("pkill netperf", params, ignore_status=True)
+        process.run("systemctl start firewalld", shell=True, verbose=True)
+        remote.run_remote_cmd("systemctl start firewalld", params)
+
+    vm_name = params.get("migrate_main_vm")
+    server_ip = params.get("server_ip")
+    server_user = params.get("server_user", "root")
+    server_pwd = params.get("server_pwd")
+    netperf_port = params.get("netperf_port")
```

Ruff (0.14.8)

Local variable `netperf_port` is assigned to but never used

Remove assignment to unused variable `netperf_port`

(F841)